src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -545,21 +586,27 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
                 apiClient.getItem(apiClient.getCurrentUserId(), nowPlayingItem.Id).then(function (item) {
                     var userData = item.UserData || {};
                     var likes = userData.Likes == null ? '' : userData.Likes;
-                    var contextButton = document.querySelector('.btnToggleContextMenu');
-                    var options = {
-                        play: false,
-                        queue: false,
-                        positionTo: contextButton
-                    };
-                    nowPlayingUserData.innerHTML = '<button is="emby-ratingbutton" type="button" class="listItemButton mediaButton paper-icon-button-light" data-id="' + item.Id + '" data-serverid="' + item.ServerId + '" data-itemtype="' + item.Type + '" data-likes="' + likes + '" data-isfavorite="' + (userData.IsFavorite) + '"><span class="material-icons favorite"></span></button>';
-                    apiClient.getCurrentUser().then(function(user) {
-                        contextButton.addEventListener('click', function () {
-                            itemContextMenu.show(Object.assign({
-                                item: item,
-                                user: user
-                            }, options ));
+                    if (!layoutManager.mobile) {
+                        let contextButton = document.querySelector('.nowPlayingBar').querySelector('.btnToggleContextMenu');
+                        // We remove the previous event listener by replacing the item in each update event
+                        let contextButtonClone = contextButton.cloneNode(true);
+                        contextButton.parentNode.replaceChild(contextButtonClone, contextButton);
+                        contextButton = document.querySelector('.nowPlayingBar').querySelector('.btnToggleContextMenu');
```

Comment:
@dmitrylyzo Here seems to work, but not in ``remotecontrol.js``. I guess it's because we reconstruct the whole DOM in the miniplayer. I'm going to leave them as I did for the sake of consistency.

I plan on revisit the players later, so this might be changed in the future.